### PR TITLE
RELATED: RAIL-564 Added onLegendReady calback

### PR DESCRIPTION
**Pull request by vojty**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/896/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/730/ : SUCCESS in 3m 13s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/203/ : SUCCESS in 1m 49s

File: src/interfaces/Events.ts [hunk 1]
```diff
@@ -9,6 +9,18 @@ export type OnError = (error: object) => void;
 export type OnLoadingChanged = (loadingState: ILoadingState) => void;
 export type OnFiredDrillEvent = (param: IDrillEvent) => void | boolean;
 
+export interface ILegendItem {
+    name: string;
+    color: string; // hex or RGB, can be used directly in CSS style
+    onClick: Function; // toggle to show/hide serie in chart
```
Comment (vaclavbohac) [inline src/interfaces/Events.ts]: Is it possible to be more specific about the function?

Is it `(): void`?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/904/ : SUCCESS in 1m 01s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/738/ : SUCCESS in 1m 09s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/211/ : SUCCESS in 35s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/919/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/749/ : FAILURE in 3m 48s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/222/ : SUCCESS in 36s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/927/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/756/ : SUCCESS in 4m 13s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/229/ : SUCCESS in 1m 42s

**Comment by vojty** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/251/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/931/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/254/ : UNSTABLE in 4m 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/933/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/760/ : SUCCESS in 2m 50s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/233/ : SUCCESS in 2m 51s

**Comment by vojty** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/252/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/934/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/255/ : UNSTABLE in 5m 31s

**Comment by vaclavbohac** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/939/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/765/ : SUCCESS in 2m 36s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/238/ : SUCCESS in 1m 43s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/253/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/940/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/256/ : UNSTABLE in 3m 22s

**Comment by vaclavbohac** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/942/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/257/ : FAILURE in 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/941/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/766/ : SUCCESS in 2m 45s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/239/ : SUCCESS in 1m 41s

**Comment by vaclavbohac** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/254/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/943/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/258/ : SUCCESS in 3m 01s

**Comment by vojty** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/948/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/771/ : SUCCESS in 2m 34s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/244/ : SUCCESS in 25s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/257/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/949/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/260/ : SUCCESS in 4m 49s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-merge-compliance-check/193/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/772/ : SUCCESS in 1m 09s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/258/ : SUCCESS in 4m 00s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/245/ : SUCCESS in 33s

**Pull request merged**